project/src/main/java/edu/depaul/cdm/se452/project/model/Student.java

```diff
@@ -36,17 +36,37 @@
 	private int gpa;
 	
 	//@Column
-	private String firstName;
+	private String first_name;
 	
 	//@Column
-	private String lastName;
+	private String last_name;
 	
 	//@Column
 	private String email;
+	
+	public int getStudent_id() {
+		return student_id;
+	}
+	
+	public String getMajor() {
+		return major;
+	}
+	public int getGpa() {
+		return gpa;
+	}
+	public String getFirst_name() {
+		return first_name;
+	}
+	public String getLast_name() {
+		return last_name;
+	}
+	public String getEmail() {
+		return email;
+	}
 		
-	@JoinColumn
-	@OneToOne(cascade = CascadeType.ALL)
-	private Course course;
+	//@JoinColumn

```

Comment:
What is the purpose of commenting this out?

Reply:
this may have been by accident